src/store/login/login.reducer.ts

```diff
@@ -0,0 +1,245 @@
+import {
+  LoginActionTypes,
+  LoginAction,
+  WalletName,
+  AccountData,
+} from "src/store/login/login.actions";
+import { AsyncTask } from "src/utils/types";
+// domain
+import { HermezWallet } from "src/domain/hermez";
+import { AuthSignatures } from "src/domain/local-storage";
+import { getAuthSignatures } from "src/persistence/local-storage";
+
+export interface LoginState {
+  step: Step;
+  // ToDo: This state is never set. The actions do not exist.
+  // networkNameTask: AsyncTask<string, string>;
```

Comment:
If we don't use it anymore, maybe it can be removed? Always happy to remove things hehe

Reply:
Yeah me too, I was afraid to remove it...

Reply:
Removed